src/it/xerces/src/test/java/com/jcabi/xerces/XercesSampleTest.java

```diff
@@ -56,6 +57,10 @@
      */
     @Test
     public void validatesXmlForSchemaValidity() throws Exception {
+        final int zero = 0;
```

Comment:
@tmjee this would look correct if you would give meaningful names to these variables. `zero` is not a good name. as well as `fifty` is not a good name either. BTW, checkstyle doesn't complain about 0, 1, -1.